web/src/Statusbar.test.tsx

```diff
@@ -185,6 +192,32 @@ describe("StatusBar", () => {
             alertsUrl="/alerts"
             runningVersion={devRunningVersion}
             latestVersion={latestVersion}
+            checkVersion={true}
+          />
+        </MemoryRouter>
+      )
+      .toJSON()
+
+    expect(tree).toMatchSnapshot()
+  })
+
+  it("does not render an upgrade badge when the version is out of date if version check is disabled", () => {
```

Comment:
my understanding is that we're trying to move away from snapshot tests and towards assertions? would prefer this as:
```
const root = mount(...)
let element = root.find(".UpgradeBadge")
expect(element).toHaveLength(0)
```
or whatever it'd be